dso-l1/src/main/java/com/tc/config/schema/setup/L1ConfigurationSetupManager.java

```diff
@@ -39,4 +40,6 @@
   L2ConfigForL1 l2Config();
 
   SecurityInfo getSecurityInfo();
+  
+  Map<String, String> getOverrideTCProperties();
```

Comment:
Yes.  This is an old feature from when the clients had the server's tc-config.  I left it so that in the future, if we wanted a new way to add properties on the client, the hook would still be there.